app/standalone.go

```diff
@@ -44,7 +48,7 @@ type standaloneData struct {
 // This will be run manually from command line ONLY
 func DoStandaloneInstall(device *dev.DeviceManager, updateURI string,
```

Comment:
Suggestion:
It could maybe be time to factor this call out into accepting a struct, which has the standard values set already.

So something like:
```
type doStandaloneInstallArgs struct {
    rebootExitCode bool
   ...
}
```
If so, add this in a separate commit, so that the other reviewers can easily chime in their opinions without creating a lot more work for you :)

Reply:
I feel like this rework is outside the scope of my change. I'll leave this one to the pros, if you don't mind. :)

Reply:
That is just fine :)